packages/odyssey-react-mui/src/components/Link/Link.tsx

```diff
@@ -24,7 +24,7 @@ export const Link = forwardRef<HTMLLinkElement | HTMLAnchorElement, LinkProps>(
     const { icon, children, target } = props;
     return (
       <MuiLink {...props}>
-        <span className="Link-icon">{icon}</span>
+        {icon && <span className="Link-icon">{icon}</span>}
```

Comment:
Applitools wouldn't pass without this fix. We were referencing MUI directly rather than Odyssey in some places, so when I imported everything from Odyssey, it caused this inconsistency.